jumpscale/packages/weblibs/package.py

```diff
@@ -11,7 +11,15 @@ def install(self):
         """Called when package is added
         """
         if not j.sals.fs.exists(j.sals.fs.join_paths(self.path, "jumpscaleX_weblibs")):
-            j.tools.git.clone_repo(url=self.url, dest=self.path, branch_or_tag=self.branch)
+            retries = 5
+            while retries:
+                try:
+                    j.tools.git.clone_repo(url=self.url, dest=self.path, branch_or_tag=self.branch)
```

Comment:
can make a shallow cloning to reduce cloning size depth=1 for example

Reply:
added, thanks